ocean_data_parser/read/vocabularies/seabird_variable_attributes.json

```diff
@@ -2527,8 +2539,9 @@
         "comments": "1st sensor"
     },
     "sal11": {
-        "long_name": "Salinity, Practical, 2",
+        "long_name": "Salinity, Practical, 2 TEST",
```

Comment:
Not sure why TEST was added

Reply:
This should be a mistake from my part. I was trying something and forgot to remove TEST. 
You can remove it.

Reply:
Ok I removed the TEST